src/Player.cpp

```diff
@@ -272,6 +272,44 @@ void Player::AddStoreItemToInventory(int type) {
 	}
 }
 
+bool Player::RemoveStoreItemFromInventory(int type) {
+	// Removes sold items from the inventory.
+	switch (type)
+	{
+	case ITEMTYPE::ARROWS:
+		if (arrows >= 5)
+		{
+			arrows -= 5;
```

Comment:
Maybe include `int amount` as a parameter to avoid hardcoding the item amount here? This would have to be changed in `AddStoreItemToInventory()` as well.

Reply:
I modified the `AddStoreItemToInventory()` function to take an arbitrary amount (similar for selling).